packages/body/src/sass/_module.scss

```diff
@@ -69,8 +69,7 @@
 	}
 
 	mark {
-		padding: 0 2px;
-		border-radius: $uikit-border-radius;
+		box-shadow: 0 0 1px $uikit-box-shadow-colour;
```

Comment:
Removed it because it caused the whitespace to look odd when that `mark` is selected.